src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -202,13 +202,13 @@ import Involucro from '../components/Involucro.astro';
 </Involucro>
 ```
 
-Este padrão é a base de um [componente de layout Astro](/pt-br/basics/layouts/): uma página inteira de conteúdo HTML pode ser "envolta" com tags `<AlgumComponenteDeLayout></AlgumComponenteDeLayout>` e enviadas até o componente para ser renderizada dentro de elementos comuns da página definidos lá.
+Este padrão é a base de um [componente de layout Astro](/pt-br/basics/layouts/): uma página inteira de conteúdo HTML pode ser "envolta" com tags `<AlgumComponenteDeLayout></AlgumComponenteDeLayout>` e enviadas até o componente para serem renderizadas dentro de elementos comuns da página definidos nele.
```

Comment:
Fix noun agreement and use "nele" instead of "lá" because "lá" is ambiguous as to what it's referring to - either the page or the component.